BTrees/BTreeTemplate.c

```diff
@@ -219,6 +219,8 @@ BTree_check(BTree *self)
     return result;
 }
 
+#define _GET_KEY_ERROR 1
+#define _GET_ALL_ERROR 0
```

Comment:
Sure. I'm open to suggestions!

My process here was:
- The param was originally named something like `ignore_def_cmp`, before I realized that the int-trees *also* raised a TypeError, just not the default-comparison type error.
- I had updated all the `_BTree_get` calls to add the param. But having two "boolean" params at the end of the call, so it looked like `_BTreeGet(self, key, 0, 1)` was pretty unreadable.
- Thinking about the callers, they either want `_BTree_get` to raise *all* lookup-related errors, or just a KeyError. Hence the names of the params, so now we have `_BTree_get(self, key, 0, _GET_KEY_ERROR)`
- Then I realized the thing about integer trees and changed the param name...which I'm just now realizing is a bit confusing because we're ignoring a TypeError *on* the key...